invenio_mail/tasks.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
```

Comment:
ping @bruno314 - what is the name for entrypoint that should specify this tasks module?